drivers/modem/ublox-sara-r4.c

```diff
@@ -380,9 +380,6 @@ static ssize_t send_socket_data(void *obj,
 	/* Wait for prompt '@' */
 	k_sem_take(&mdata.sem_prompt, K_FOREVER);
 
-	/* slight pause per spec so that @ prompt is received */
-	k_sleep(MDM_PROMPT_CMD_DELAY);
```

Comment:
Please remove:
```
#define MDM_PROMPT_CMD_DELAY		K_MSEC(50)
```
as it is no longer used.

Reply:
Be careful as this was added to allow the U2 version as per the manufacturer recommendation.

Reply:
Obsolete comment; see updated PR